app/Models/Post.php

```diff
@@ -336,21 +336,12 @@ public function siblings()
     }
 
     /**
-     * Transform the post model for Blink.
+     * Transform this post for Customer.io.
      *
      * @return array
      */
     public function toCustomerIoPayload()
     {
-        // Blink expects quantity to be a number.
-        $quantity = $this->quantity === null ? 0 : $this->quantity;
```

Comment:
We no longer need to play by Blink's validation rules, but this is also something we can handle by simply casting the `$this->quantity` to an integer (since `(int) null` becomes `0`)!